plugins/techdocs-react/package.json

```diff
@@ -42,7 +42,7 @@
     "@material-ui/core": "^4.12.2",
     "@material-ui/lab": "4.0.0-alpha.57",
     "@material-ui/styles": "^4.11.0",
-    "jss": "~10.9.0",
+    "jss": "^10.5.1",
```

Comment:
This is here because of the `techdocs` package?

Reply:
@aitherios see [here](https://github.com/backstage/backstage/pull/15855/files#diff-a768fbf60c0ae56ed8e72f6d5c6d9d5ff24fdc03484f0f8dadf164a13f04c2a5), the error mentioned in the changeset happens when I try to update the API reports for this package, and I need to update the report because I'm exporting a new extension. @backstage/techdocs-core do you have any recommendations?

Reply:
@Rugvip also seems to be a breakage in a transitive upstream dependency.

Reply:
It seems to be fixed, closing this.